app/components/syntax-highlighted-diff.ts

```diff
@@ -36,12 +64,6 @@ export default class SyntaxHighlightedDiffComponent extends Component<Signature>
   static LINES_AROUND_CHANGED_CHUNK = 3;
   static MIN_LINES_BETWEEN_CHUNKS_BEFORE_COLLAPSING = 4;
 
-  constructor(owner: unknown, args: Signature['Args']) {
-    super(owner, args);
-
-    this.highlightCode.perform();
-  }
-
```

Comment:
This is unnecessary, as invoking `highlightCode` at this point results into the task exiting immediately, because `#isDarkMode` is still `undefined`

Reply:
Now using `did-insert` to call `hightlightCode` instead of the constructor